aikau/src/main/resources/alfresco/forms/controls/css/MultiSelect.css

```diff
@@ -168,11 +168,11 @@
    }
    &--has-error {
       border-color: #c00;
-      outline: 1px solid #faa;
+      box-shadow: 0 0 1px 1px #ffc4c4;
```

Comment:
You can leave this for now - there is already another JIRA issue for addressing LESS changes.